### PR TITLE
leverage libnodejs

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
fixes https://github.com/paketo-buildpacks/libnodejs/issues/17

## Summary

Make use of `libnodejs` for some common functionality.

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM assuming the tests pass

**Comment by c0d1ngm0nk3y** [comment]
@paketo-buildpacks/nodejs-maintainers Since all tests passed. Is there a chance to get this merged? It is basically only reusing functionality from `libnodejs`.

**Comment by c0d1ngm0nk3y** [comment]
I rebased and resolved the conflict. Any chance on getting this merged?

File: init_test.go
Comment (thitch97) [inline init_test.go:11]: For posterity, what's the benefit of removing `spec.Parallel`?

**Review by thitch97 — APPROVED** [review]

**Pull request merged**
